composer.json

```diff
@@ -16,7 +16,7 @@
         "phergie/phergie-irc-connection": "~2.0",
         "react/event-loop": "~0.4.0",
         "react/stream": "~0.4.2",
-        "react/socket-client": "~0.4.2",
+        "react/socket-client": "~0.5.0",
```

Comment:
I'd recommend changing `~0.5.0` to `^0.5`. It appears that 0.5.3 is available, while this version restriction is only allowing 0.5.1 to be installed.